src/components/Thought.tsx

```diff
@@ -149,23 +150,6 @@ const ThoughtContainer = ({
     return !isExpandedHoverTopPath && !!cursor && equalPath(cursorGrandparent, path)
   })
 
-  // Note: If the thought is the active expand hover top path then it should be treated as a cursor parent. It is because the current implementation allows tree to unfold visually starting from cursor parent.
-  const isCursorParent = useSelector(state => {
```

Comment:
The `isCursorParent` definition seems to have been removed completely. Was that a mistake or am I missing something? Thanks!

Reply:
Yes sorry about that, I'm not sure how that wound up in there, considering it wasn't even in the right commit